OWNERS.md

```diff
@@ -5,8 +5,6 @@
 This page lists all active maintainers and their areas of expertise. This can be used for
 routing PRs, questions, etc. to the right place.
 
-* Alan Chiu ([buildbreaker](https://github.com/buildbreaker)) (achiu@lyft.com)
```

Comment:
Please move to the emeritus section below.

Reply:
done